syncer/util.go

```diff
@@ -47,41 +47,41 @@ func binlogTypeToString(binlogType BinlogType) string {
 	}
 }
 
-// tableNameForDML gets table name from INSERT/UPDATE/DELETE statement.
-func tableNameForDML(dml ast.DMLNode) (schema, table string, err error) {
+// getTableByDML gets table from INSERT/UPDATE/DELETE statement.
+func getTableByDML(dml ast.DMLNode) (*filter.Table, error) {
 	switch stmt := dml.(type) {
 	case *ast.InsertStmt:
 		if stmt.Table == nil || stmt.Table.TableRefs == nil || stmt.Table.TableRefs.Left == nil {
-			return "", "", terror.ErrSyncUnitInvalidTableName.Generate(fmt.Sprintf("INSERT statement %s not valid", stmt.Text()))
+			return &filter.Table{}, terror.ErrSyncUnitInvalidTableName.Generate(fmt.Sprintf("INSERT statement %s not valid", stmt.Text()))
```

Comment:
```suggestion
			return nil, terror.ErrSyncUnitInvalidTableName.Generate(fmt.Sprintf("INSERT statement %s not valid", stmt.Text()))
```

Reply:
If err is not nil, return nil is enough. Please also replace all the following ones.